ClassCastException on InventoryGui Version 1.6.5

### What is happening?

When interacting with an item that opens an inventory, this error is given:

```
[19:45:25 ERROR]: Could not pass event PlayerInteractEvent to LMGHub v1.0.0
java.lang.ClassCastException: Cannot cast [Ljava.lang.Object; to org.bukkit.inventory.InventoryView
at java.base/java.lang.Class.cast(Class.java:4067) ~[?:?]
at LMGHub-1.0.0-all.jar/de.themoep.inventorygui.GuiView.invoke(GuiView.java:109) ~[LMGHub-1.0.0-all.jar:?]
at LMGHub-1.0.0-all.jar/de.themoep.inventorygui.GuiView.getType(GuiView.java:59) ~[LMGHub-1.0.0-all.jar:?]
at LMGHub-1.0.0-all.jar/de.themoep.inventorygui.InventoryGui.show(InventoryGui.java:598) ~[LMGHub-1.0.0-all.jar:?]
at LMGHub-1.0.0-all.jar/de.themoep.inventorygui.InventoryGui.show(InventoryGui.java:586) ~[LMGHub-1.0.0-all.jar:?]
at LMGHub-1.0.0-all.jar/gg.leg.hub.LMGHubPlugin.onPlayerInteract(LMGHubPlugin.java:183) ~[LMGHub-1.0.0-all.jar:?]
at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:71) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:526) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:487) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:483) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:479) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.server.network.ServerGamePacketListenerImpl.handleUseItem(ServerGamePacketListenerImpl.java:2104) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.network.protocol.game.ServerboundUseItemPacket.handle(ServerboundUseItemPacket.java:48) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.network.protocol.game.ServerboundUseItemPacket.handle(ServerboundUseItemPacket.java:9) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:29) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:155) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1449) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:176) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:129) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1429) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1423) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:139) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1380) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1388) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1265) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:310) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
```

### What did you expect to happen?

When I interact with the item, the inventory should open as normal, with no errors.

### Version

1.6.5-SNAPSHOT

### Config

```YAML
[No config file]
```

### Server/system Version

1.21.7-8-main@6e021c8

### Log

```shell
[19:45:05] [ServerMain/INFO]: [bootstrap] Running Java 21 (OpenJDK 64-Bit Server VM 21.0.7+6-LTS; Eclipse Adoptium Temurin-21.0.7+6) on Linux 6.1.0-37-cloud-amd64 (amd64)
[19:45:05] [ServerMain/INFO]: [bootstrap] Loading Paper 1.21.7-8-main@6e021c8 (2025-06-30T18:06:02Z) for Minecraft 1.21.7
[19:45:05] [ServerMain/INFO]: [PluginInitializerManager] Initializing plugins...
[19:45:05] [Paper Plugin Remapper Thread - 2/INFO]: [PluginRemapper] Remapping plugin 'plugins/LMGHub-1.0.0-all.jar'...
[19:45:06] [Paper Plugin Remapper Thread - 2/INFO]: [PluginRemapper] Done remapping plugin 'plugins/LMGHub-1.0.0-all.jar' in 1177ms.
[19:45:07] [ServerMain/INFO]: [PluginInitializerManager] Initialized 7 plugins
[19:45:07] [ServerMain/INFO]: [PluginInitializerManager] Paper plugins (3):
 - FancyHolograms (2.6.0.1**), FancyNpcs (2.6.0.2**), SignedVelocity (1.3.0)
[19:45:07] [ServerMain/INFO]: [PluginInitializerManager] Bukkit plugins (4):
 - FastAsyncWorldEdit (2.13.1-SNAPSHOT-1122;9d18596), HeadDatabase (4.20.4), LMGHub (1.0.0), WorldGuard (7.0.13-beta-2+5c4848b)
[19:45:11] [ServerMain/WARN]: You specified a resource pack without providing a sha1 hash. Pack will be updated on the client only if you change the name of the pack.
[19:45:11] [ServerMain/WARN]: resource-pack-id missing, using default of 5519cece-6fec-3b5f-bf54-ec19f0873e58
[19:45:11] [ServerMain/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[19:45:12] [ServerMain/INFO]: Loaded 1407 recipes
[19:45:12] [ServerMain/INFO]: Loaded 1520 advancements
[19:45:12] [ServerMain/INFO]: [ca.spottedleaf.dataconverter.minecraft.datatypes.MCTypeRegistry] Initialising converters for DataConverter...
[19:45:13] [ServerMain/INFO]: [ca.spottedleaf.dataconverter.minecraft.datatypes.MCTypeRegistry] Finished initialising converters for DataConverter in 292.4ms
[19:45:13] [Server thread/INFO]: Starting minecraft server version 1.21.7
[19:45:13] [Server thread/INFO]: Loading properties
[19:45:13] [Server thread/INFO]: This server is running Paper version 1.21.7-8-main@6e021c8 (2025-06-30T18:06:02Z) (Implementing API version 1.21.7-R0.1-SNAPSHOT)
[19:45:13] [Server thread/INFO]: [spark] This server bundles the spark profiler. For more information please visit https://docs.papermc.io/paper/profiling
[19:45:13] [Server thread/INFO]: Server Ping Player Sample Count: 12
[19:45:13] [Server thread/INFO]: Using 4 threads for Netty based IO
[19:45:13] [Server thread/INFO]: [MoonriseCommon] Paper is using 1 worker threads, 1 I/O threads
[19:45:13] [Server thread/INFO]: [ChunkTaskScheduler] Chunk system is using population gen parallelism: true
[19:45:13] [Server thread/INFO]: Default game type: ADVENTURE
[19:45:13] [Server thread/INFO]: Generating keypair
[19:45:13] [Server thread/INFO]: Starting Minecraft server on 0.0.*.*:25574
[19:45:13] [Server thread/INFO]: Using epoll channel type
[19:45:13] [Server thread/INFO]: Paper: Using libdeflate (Linux x86_64) compression from Velocity.
[19:45:13] [Server thread/INFO]: Paper: Using OpenSSL 3.x.x (Linux x86_64) cipher from Velocity.
[19:45:13] [Server thread/INFO]: [FancyNpcs] Loading server plugin FancyNpcs v2.6.0.2**
[19:45:13] [Server thread/INFO]: [FastAsyncWorldEdit] Loading server plugin FastAsyncWorldEdit v2.13.1-SNAPSHOT-1122;9d18596
[19:45:14] [Server thread/INFO]: Got request to register class com.sk89q.worldedit.bukkit.BukkitServerInterface with WorldEdit [com.sk89q.worldedit.extension.platform.PlatformManager@6c813a33]
[19:45:14] [Server thread/INFO]: [LMGHub] Loading server plugin LMGHub v1.0.0
[19:45:14] [Server thread/INFO]: [HeadDatabase] Loading server plugin HeadDatabase v4.20.4
[19:45:14] [Server thread/INFO]: [FancyHolograms] Loading server plugin FancyHolograms v2.6.0.1**
[19:45:14] [Server thread/INFO]: [WorldGuard] Loading server plugin WorldGuard v7.0.13-beta-2+5c4848b
[19:45:14] [Server thread/INFO]: [SignedVelocity] Loading server plugin SignedVelocity v1.3.0
[19:45:14] [FancyLogger/INFO]: [FancyHolograms] (Server thread) INFO: Successfully loaded FancyHolograms version 2.6.0.1**
[19:45:14] [Server thread/INFO]: Server permissions file permissions.yml is empty, ignoring it
[19:45:14] [Server thread/INFO]: [FastAsyncWorldEdit] Enabling FastAsyncWorldEdit v2.13.1-SNAPSHOT-1122;9d18596
[19:45:14] [Server thread/INFO]: [com.fastasyncworldedit.core.Fawe] LZ4 Compression Binding loaded successfully
[19:45:14] [Server thread/INFO]: [com.fastasyncworldedit.core.Fawe] ZSTD Compression Binding loaded successfully
[19:45:14] [Server thread/INFO]: Registering commands with com.sk89q.worldedit.bukkit.BukkitServerInterface
[19:45:14] [Server thread/INFO]: WEPIF: Using the Bukkit Permissions API.
[19:45:15] [Server thread/WARN]: 
**********************************************
** This FastAsyncWorldEdit version does not fully support your version of Bukkit.
** You can fix this by:
** - Updating your server version (Check /version to see how many versions you are behind)
** - Updating FAWE
**
** When working with blocks or undoing, chests will be empty, signs
** will be blank, and so on. There will be no support for entity
** and block property-related functions.
**********************************************

[19:45:15] [Server thread/WARN]: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
[19:45:15] [Server thread/WARN]: 	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar//com.sk89q.worldedit.world.block.BlockTypesCache.<clinit>(BlockTypesCache.java:237)
[19:45:15] [Server thread/WARN]: 	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar//com.sk89q.worldedit.world.block.BlockTypes.init(BlockTypes.java:2281)
[19:45:15] [Server thread/WARN]: 	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar//com.sk89q.worldedit.world.block.BlockTypes.<clinit>(BlockTypes.java:48)
[19:45:15] [Server thread/WARN]: 	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar//com.sk89q.worldedit.world.block.BlockState.get(BlockState.java:156)
[19:45:15] [Server thread/WARN]: 	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar//com.sk89q.worldedit.world.block.BlockState.get(BlockState.java:132)
[19:45:15] [Server thread/WARN]: 	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar//com.sk89q.worldedit.world.registry.LegacyMapper.loadFromResource(LegacyMapper.java:124)
[19:45:15] [Server thread/WARN]: 	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar//com.sk89q.worldedit.world.registry.LegacyMapper.<init>(LegacyMapper.java:84)
[19:45:15] [Server thread/WARN]: 	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar//com.sk89q.worldedit.world.registry.LegacyMapper.getInstance(LegacyMapper.java:329)
[19:45:15] [Server thread/WARN]: 	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar//com.sk89q.worldedit.WorldEdit.loadMappings(WorldEdit.java:428)
[19:45:15] [Server thread/WARN]: 	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar//com.sk89q.worldedit.bukkit.WorldEditPlugin.setupPreWorldData(WorldEditPlugin.java:294)
[19:45:15] [Server thread/WARN]: 	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar//com.sk89q.worldedit.bukkit.WorldEditPlugin.onEnable(WorldEditPlugin.java:264)
[19:45:15] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:280)
[19:45:15] [Server thread/WARN]: 	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202)
[19:45:15] [Server thread/WARN]: 	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109)
[19:45:15] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:520)
[19:45:15] [Server thread/WARN]: 	at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:651)
[19:45:15] [Server thread/WARN]: 	at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:607)
[19:45:15] [Server thread/WARN]: 	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:236)
[19:45:15] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1164)
[19:45:15] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:310)
[19:45:15] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:1583)
[19:45:15] [Server thread/WARN]: Failed to load the built-in legacy id registry
java.lang.ExceptionInInitializerError: null
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.block.BlockTypes.init(BlockTypes.java:2281) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.block.BlockTypes.<clinit>(BlockTypes.java:48) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.block.BlockState.get(BlockState.java:156) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.block.BlockState.get(BlockState.java:132) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.registry.LegacyMapper.loadFromResource(LegacyMapper.java:124) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.registry.LegacyMapper.<init>(LegacyMapper.java:84) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.registry.LegacyMapper.getInstance(LegacyMapper.java:329) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.WorldEdit.loadMappings(WorldEdit.java:428) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.bukkit.WorldEditPlugin.setupPreWorldData(WorldEditPlugin.java:294) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.bukkit.WorldEditPlugin.onEnable(WorldEditPlugin.java:264) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:280) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:520) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:651) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:607) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:236) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1164) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:310) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.RuntimeException: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.block.BlockTypesCache.<clinit>(BlockTypesCache.java:278) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	... 20 more
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.block.BlockTypesCache.<clinit>(BlockTypesCache.java:237) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	... 20 more
[19:45:15] [Server thread/WARN]: **** SERVER IS RUNNING IN OFFLINE/INSECURE MODE!
[19:45:15] [Server thread/WARN]: The server will make no attempt to authenticate usernames. Beware.
[19:45:15] [Server thread/WARN]: Whilst this makes it possible to use Velocity, unless access to your server is properly restricted, it also opens up the ability for hackers to connect with any username they choose.
[19:45:15] [Server thread/WARN]: Please see https://docs.papermc.io/velocity/security for further information.
[19:45:15] [Server thread/WARN]: To change this, set "online-mode" to "true" in the server.properties file.
[19:45:15] [Server thread/INFO]: Preparing level "LegundoSprings"
[19:45:16] [Server thread/WARN]: Failed to load biomes via adapter (not present). Will load via bukkit
[19:45:16] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[19:45:16] [Server thread/INFO]: Preparing spawn area: 0%
[19:45:17] [Server thread/INFO]: Preparing spawn area: 6%
[19:45:17] [Server thread/INFO]: Time elapsed: 988 ms
[19:45:17] [Server thread/INFO]: Preparing start region for dimension minecraft:the_nether
[19:45:17] [Server thread/INFO]: Preparing spawn area: 0%
[19:45:17] [Server thread/INFO]: Time elapsed: 178 ms
[19:45:17] [Server thread/INFO]: Preparing start region for dimension minecraft:the_end
[19:45:17] [Server thread/INFO]: Preparing spawn area: 0%
[19:45:17] [Server thread/INFO]: Time elapsed: 93 ms
[19:45:17] [Server thread/INFO]: [FancyNpcs] Enabling FancyNpcs v2.6.0.2**
[19:45:17] [Server thread/WARN]: [org.mineskin.MineSkinClient] Creating MineSkinClient without API key
[19:45:18] [FancyLogger/INFO]: [FancyNpcs] (Server thread) INFO: FancyNpcs (2.6.0.2**) has been enabled.
[19:45:18] [Server thread/INFO]: [LMGHub] Enabling LMGHub v1.0.0
[19:45:18] [Server thread/INFO]: Loading LMG Core version v0.2
[19:45:18] [Server thread/INFO]: [LMGHub] Registering events...
[19:45:18] [Server thread/INFO]: [LMGHub] Registering configs...
[19:45:18] [Server thread/INFO]: [com.zaxxer.hikari.HikariDataSource] HikariPool-1 - Starting...
[19:45:18] [Server thread/INFO]: [com.zaxxer.hikari.pool.HikariPool] HikariPool-1 - Added connection com.mysql.cj.jdbc.ConnectionImpl@736bb314
[19:45:18] [Server thread/INFO]: [com.zaxxer.hikari.HikariDataSource] HikariPool-1 - Start completed.
[19:45:18] [Server thread/INFO]: [LMGHub] [Leg.GG Matchmaking] > Refreshing cosmetic items library.
[19:45:18] [Server thread/INFO]: [LMGHub] [Leg.GG Matchmaking] > Loaded 60 cosmetic items to inventory
[19:45:18] [Server thread/INFO]: [LMGHub] [Leg.GG Matchmaking] > >> 25 items in HEAD category.
[19:45:18] [Server thread/INFO]: [LMGHub] [Leg.GG Matchmaking] > >> 35 items in OFFHAND category.
[19:45:18] [Server thread/INFO]: [LMGHub] Registering commands...
[19:45:18] [Server thread/INFO]: [HeadDatabase] Enabling HeadDatabase v4.20.4
[19:45:19] [Server thread/INFO]: [HeadDatabase] §bUsing default §3"en_US.lang" §bcreated by §6Arcaniax
[19:45:19] [Server thread/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[19:45:19] [Server thread/WARN]: [HeadDatabase] Economy was not loaded, some features will be disabled!
[19:45:19] [Server thread/INFO]: [FancyHolograms] Enabling FancyHolograms v2.6.0.1**
[19:45:19] [FancyLogger/INFO]: [FancyHolograms] (Server thread) INFO: Successfully enabled FancyHolograms version 2.6.0.1**
[19:45:19] [Server thread/INFO]: [WorldGuard] Enabling WorldGuard v7.0.13-beta-2+5c4848b
[19:45:20] [Server thread/INFO]: [WorldGuard] (LegundoSprings) TNT ignition is PERMITTED.
[19:45:20] [Server thread/INFO]: [WorldGuard] (LegundoSprings) Lighters are PERMITTED.
[19:45:20] [Server thread/INFO]: [WorldGuard] (LegundoSprings) Lava fire is PERMITTED.
[19:45:20] [Server thread/INFO]: [WorldGuard] (LegundoSprings) Fire spread is UNRESTRICTED.
[19:45:20] [Server thread/INFO]: [WorldGuard] Loaded configuration for world 'LegundoSprings'
[19:45:20] [Server thread/INFO]: [WorldGuard] (LegundoSprings_nether) TNT ignition is PERMITTED.
[19:45:20] [Server thread/INFO]: [WorldGuard] (LegundoSprings_nether) Lighters are PERMITTED.
[19:45:20] [Server thread/INFO]: [WorldGuard] (LegundoSprings_nether) Lava fire is PERMITTED.
[19:45:20] [Server thread/INFO]: [WorldGuard] (LegundoSprings_nether) Fire spread is UNRESTRICTED.
[19:45:20] [Server thread/INFO]: [WorldGuard] Loaded configuration for world 'LegundoSprings_nether'
[19:45:20] [Server thread/INFO]: [WorldGuard] (LegundoSprings_the_end) TNT ignition is PERMITTED.
[19:45:20] [Server thread/INFO]: [WorldGuard] (LegundoSprings_the_end) Lighters are PERMITTED.
[19:45:20] [Server thread/INFO]: [WorldGuard] (LegundoSprings_the_end) Lava fire is PERMITTED.
[19:45:20] [Server thread/INFO]: [WorldGuard] (LegundoSprings_the_end) Fire spread is UNRESTRICTED.
[19:45:20] [Server thread/INFO]: [WorldGuard] Loaded configuration for world 'LegundoSprings_the_end'
[19:45:20] [Server thread/INFO]: [WorldGuard] Loading region data...
[19:45:20] [Server thread/INFO]: [SignedVelocity] Enabling SignedVelocity v1.3.0
[19:45:20] [Server thread/WARN]: [SignedVelocity] ------------------------------
[19:45:20] [Server thread/WARN]: [SignedVelocity] The following plugins have listener in legacy org.bukkit.event.player.AsyncPlayerChatEvent event: HeadDatabase, WorldGuard.
[19:45:20] [Server thread/WARN]: [SignedVelocity] This may negatively affect the functionality of SignedVelocity,
[19:45:20] [Server thread/WARN]: [SignedVelocity] please report to the author to use Paper's AsyncChatEvent and/or AsyncChatDecorateEvent.
[19:45:20] [Server thread/WARN]: [SignedVelocity] ------------------------------
[19:45:20] [Server thread/INFO]: [spark] Starting background profiler...
[19:45:20] [Folia Async Scheduler Thread #0/INFO]: [HeadDatabase] Successfully loaded 69343 heads!
[19:45:21] [Folia Async Scheduler Thread #0/INFO]: [HeadDatabase] Successfully loaded 18 featured tags!
[19:45:21] [Folia Async Scheduler Thread #1/WARN]: [HeadDatabase] §cYou are using an outdated version!
[19:45:21] [Folia Async Scheduler Thread #1/WARN]: [HeadDatabase] Latest version: §a4.21.2§e. You are on version: §c4.20.4§e.
[19:45:21] [Folia Async Scheduler Thread #1/WARN]: [HeadDatabase] Update here: §bhttps://www.spigotmc.org/resources/head-database.14280/
[19:45:21] [Server thread/INFO]: Done preparing level "LegundoSprings" (5.453s)
[19:45:21] [Server thread/INFO]: Running delayed init tasks
[19:45:21] [Server thread/INFO]: [com.fastasyncworldedit.bukkit.regions.WorldGuardFeature] Plugin 'WorldGuard' found. Using it now.
[19:45:21] [Server thread/INFO]: [com.fastasyncworldedit.bukkit.FaweBukkit] Attempting to use plugin 'WorldGuard'
[19:45:21] [Server thread/INFO]: Done (16.592s)! For help, type "help"
[19:45:21] [User Authenticator #0/INFO]: UUID of player RyzechDev is fde2d418-ede8-436d-8b1a-bc9586763539
[19:45:21] [Server thread/WARN]: [HorriblePlayerLoginEventHack] 
============================================================
WARNING: Legacy PlayerLoginEvent usage detected!

This event forces an alternative player loading path that is
deprecated and will be removed in a future release.
For more information, see: https://go.papermc.io/announcement/1.21.7

Please notify the following plugin developers: [LMGHub, FancyNpcs, WorldGuard]
============================================================
[19:45:23] [Server thread/INFO]: [LMGHub] [Leg.GG Matchmaking] > Refreshing playlist status.
[19:45:23] [Server thread/INFO]: [LMGHub] [Leg.GG Matchmaking] > Refreshing player cosmetic outfit.
[19:45:23] [Server thread/INFO]: [LMGHub] [Leg.GG Matchmaking] > Refreshing player cosmetic items library for player: RyzechDev
[19:45:23] [Server thread/INFO]: RyzechDev joined the game
[19:45:23] [Server thread/INFO]: RyzechDev[/107.193.218.1**:44912] logged in with entity id 51 at ([LegundoSprings]0.5, 106.0, 16.5)
[19:45:25] [Server thread/ERROR]: Could not pass event PlayerItemHeldEvent to FastAsyncWorldEdit v2.13.1-SNAPSHOT-1122;9d18596
java.lang.NoClassDefFoundError: Could not initialize class com.sk89q.worldedit.world.block.BlockTypes
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.fastasyncworldedit.bukkit.adapter.CachedBukkitAdapter.init(CachedBukkitAdapter.java:36) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.fastasyncworldedit.bukkit.adapter.CachedBukkitAdapter.asItemType(CachedBukkitAdapter.java:58) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.bukkit.BukkitAdapter.asItemType(BukkitAdapter.java:434) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.fastasyncworldedit.bukkit.util.BukkitItemStack.<init>(BukkitItemStack.java:20) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.fastasyncworldedit.bukkit.adapter.IBukkitAdapter.adapt(IBukkitAdapter.java:253) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.bukkit.BukkitAdapter.adapt(BukkitAdapter.java:487) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.bukkit.BukkitPlayer.getItemInHand(BukkitPlayer.java:140) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.LocalSession.getTool(LocalSession.java:1239) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.fastasyncworldedit.bukkit.listener.BrushListener.onPlayerItemHoldEvent(BrushListener.java:33) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:71) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleSetCarriedItem(ServerGamePacketListenerImpl.java:2216) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.network.protocol.game.ServerboundSetCarriedItemPacket.handle(ServerboundSetCarriedItemPacket.java:33) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.network.protocol.game.ServerboundSetCarriedItemPacket.handle(ServerboundSetCarriedItemPacket.java:8) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:29) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:155) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1449) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:176) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:129) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1429) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1423) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.util.thread.BlockableEventLoop.runAllTasks(BlockableEventLoop.java:118) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1384) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1265) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:310) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.ExceptionInInitializerError [in thread "Server thread"]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.block.BlockTypes.init(BlockTypes.java:2281) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.block.BlockTypes.<clinit>(BlockTypes.java:48) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.block.BlockState.get(BlockState.java:156) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.block.BlockState.get(BlockState.java:132) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.registry.LegacyMapper.loadFromResource(LegacyMapper.java:124) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.registry.LegacyMapper.<init>(LegacyMapper.java:84) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.world.registry.LegacyMapper.getInstance(LegacyMapper.java:329) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.WorldEdit.loadMappings(WorldEdit.java:428) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.bukkit.WorldEditPlugin.setupPreWorldData(WorldEditPlugin.java:294) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.bukkit.WorldEditPlugin.onEnable(WorldEditPlugin.java:264) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:280) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:520) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:651) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:607) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:236) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1164) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	... 2 more
[19:45:25] [Server thread/ERROR]: [com.fastasyncworldedit.bukkit.FaweBukkit] Persistent Brushes Failed
java.lang.NullPointerException: null
	at com.google.gson.internal.$Gson$Preconditions.checkNotNull($Gson$Preconditions.java:47) ~[gson-2.11.0.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.fastasyncworldedit.bukkit.util.ItemUtil.<init>(ItemUtil.java:32) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.fastasyncworldedit.bukkit.FaweBukkit.getItemUtil(FaweBukkit.java:140) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.fastasyncworldedit.bukkit.util.BukkitItemStack.getNativeItem(BukkitItemStack.java:37) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.LocalSession.getTool(LocalSession.java:1247) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.LocalSession.getTool(LocalSession.java:1240) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.extension.platform.PlatformManager.handlePlayerInput(PlatformManager.java:525) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.util.eventbus.MethodHandleEventHandler.dispatch(MethodHandleEventHandler.java:51) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.util.eventbus.EventBus.dispatch(EventBus.java:193) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.util.eventbus.EventBus.post(EventBus.java:181) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.WorldEdit.handleRightClick(WorldEdit.java:757) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar/com.sk89q.worldedit.bukkit.WorldEditListener.onPlayerInteract(WorldEditListener.java:174) ~[FastAsyncWorldEdit-Paper-2.13.1-SNAPSHOT-1122.jar:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:71) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:526) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:487) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:483) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:479) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleUseItem(ServerGamePacketListenerImpl.java:2104) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.network.protocol.game.ServerboundUseItemPacket.handle(ServerboundUseItemPacket.java:48) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.network.protocol.game.ServerboundUseItemPacket.handle(ServerboundUseItemPacket.java:9) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:29) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:155) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1449) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:176) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:129) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1429) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1423) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:139) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1380) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1388) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1265) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:310) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
[19:45:25] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to LMGHub v1.0.0
java.lang.ClassCastException: Cannot cast [Ljava.lang.Object; to org.bukkit.inventory.InventoryView
	at java.base/java.lang.Class.cast(Class.java:4067) ~[?:?]
	at LMGHub-1.0.0-all.jar/de.themoep.inventorygui.GuiView.invoke(GuiView.java:109) ~[LMGHub-1.0.0-all.jar:?]
	at LMGHub-1.0.0-all.jar/de.themoep.inventorygui.GuiView.getType(GuiView.java:59) ~[LMGHub-1.0.0-all.jar:?]
	at LMGHub-1.0.0-all.jar/de.themoep.inventorygui.InventoryGui.show(InventoryGui.java:598) ~[LMGHub-1.0.0-all.jar:?]
	at LMGHub-1.0.0-all.jar/de.themoep.inventorygui.InventoryGui.show(InventoryGui.java:586) ~[LMGHub-1.0.0-all.jar:?]
	at LMGHub-1.0.0-all.jar/gg.leg.hub.LMGHubPlugin.onPlayerInteract(LMGHubPlugin.java:183) ~[LMGHub-1.0.0-all.jar:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:71) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-api-1.21.7-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:526) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:487) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:483) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:479) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleUseItem(ServerGamePacketListenerImpl.java:2104) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.network.protocol.game.ServerboundUseItemPacket.handle(ServerboundUseItemPacket.java:48) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.network.protocol.game.ServerboundUseItemPacket.handle(ServerboundUseItemPacket.java:9) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:29) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:155) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1449) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:176) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:129) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1429) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1423) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:139) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1380) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1388) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1265) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:310) ~[paper-1.21.7.jar:1.21.7-8-6e021c8]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
[19:47:52] [Server thread/INFO]: Checking version, please wait...
[19:47:52] [Thread-18/INFO]: This server is running Paper version 1.21.7-8-main@6e021c8 (2025-06-30T18:06:02Z) (Implementing API version 1.21.7-R0.1-SNAPSHOT)
You are running the latest version
Previous version: 1.21.5-114-a1b3058 (MC: 1.21.5)
```

### What other programs/plugins are you running?

FancyNpcs 2.6.0
FancyHolograms 2.6.0
SignedVelocity 1.3.0
FAWE 2.13.1-SNAPSHOT-1122;9d18596
HeadDatabase 4.20.4
WorldGuard 7.0.13-beta-2+5c4848b

### Additional context

Downgrading to InventoryGui 1.6.4-SNAPSHOT fixes the issue, 

1.6.3-SNAPSHOT and 1.6.4-SNAPSHOT both work, 1.6.5-SNAPSHOT does not work.

--- a/src/main/java/de/themoep/inventorygui/GuiView.java
+++ b/src/main/java/de/themoep/inventorygui/GuiView.java
@@ -44,69 +44,24 @@ public class GuiView {
     }
 
     public Inventory getTopInventory() {
-        return invoke(GET_TOP_INVENTORY, view);
+        return invoke(GET_TOP_INVENTORY);
     }
 
     public Inventory getBottomInventory() {
-        return invoke(GET_BOTTOM_INVENTORY, view);
+        return invoke(GET_BOTTOM_INVENTORY);
     }
 
     public HumanEntity getPlayer() {
-        return invoke(GET_PLAYER, view);
+        return invoke(GET_PLAYER);
     }
 
     public InventoryType getType() {
-        return invoke(GET_TYPE, view);
+        return invoke(GET_TYPE);
     }
 
     public void setItem(int slot, ItemStack item) {
-        invokeVoid(SET_ITEM, view, slot, item);
-    }
-
-    public ItemStack getItem(int slot) {
-        return invoke(GET_ITEM, view, slot);
-    }
-
-    public void setCursor(ItemStack item) {
-        invokeVoid(SET_CURSOR, view, item);
-    }
-
-    public ItemStack getCursor() {
-        return invoke(GET_CURSOR, view);
-    }
-
-    public Inventory getInventory(int rawSlot) {
-        return invoke(GET_INVENTORY, view, rawSlot);
-    }
-
-    public int convertSlot(int slot) {
-        return invoke(CONVERT_SLOT, view, slot);
-    }
-
-    public InventoryType.SlotType getSlotType(int slot) {
-        return invoke(GET_SLOT_TYPE, view, slot);
-    }
-
-    public void close() {
-        invokeVoid(CLOSE, view);
-    }
-
-    public int countSlots() {
-        return invoke(COUNT_SLOTS, view);
-    }
-
-    public boolean setProperty(InventoryView.Property prop, int value) {
-        return invoke(SET_PROPERTY, view, prop, value);
-    }
-
-    public String getTitle() {
-        return invoke(GET_TITLE, view);
-    }
-
-    @SuppressWarnings("unchecked")
-    private <T> T invoke(MethodHandle method, Object... args) {
         try {
-            return (T) method.invoke(args);
+            SET_ITEM.invoke(view, slot, item);
         } catch (RuntimeException e) {
             throw e;
         } catch (Throwable t) {
@@ -114,9 +69,79 @@ public class GuiView {
         }
     }
 
-    private void invokeVoid(MethodHandle method, Object... args) {
+    public ItemStack getItem(int slot) {
+        return invoke(GET_ITEM, slot);
+    }
+
+    public void setCursor(ItemStack item) {
         try {
-            method.invoke(args);
+            SET_CURSOR.invoke(view, item);
+        } catch (RuntimeException e) {
+            throw e;
+        } catch (Throwable t) {
+            throw new AssertionError(t);
+        }
+    }
+
+    public ItemStack getCursor() {
+        return invoke(GET_CURSOR);
+    }
+
+    public Inventory getInventory(int rawSlot) {
+        return invoke(GET_INVENTORY, rawSlot);
+    }
+
+    public int convertSlot(int slot) {
+        return invoke(CONVERT_SLOT, slot);
+    }
+
+    public InventoryType.SlotType getSlotType(int slot) {
+        return invoke(GET_SLOT_TYPE, slot);
+    }
+
+    public void close() {
+        try {
+            CLOSE.invoke(view);
+        } catch (RuntimeException e) {
+            throw e;
+        } catch (Throwable t) {
+            throw new AssertionError(t);
+        }
+    }
+
+    public int countSlots() {
+        return invoke(COUNT_SLOTS);
+    }
+
+    public boolean setProperty(InventoryView.Property prop, int value) {
+        try {
+            return (boolean) SET_PROPERTY.invoke(prop, value);
+        } catch (RuntimeException e) {
+            throw e;
+        } catch (Throwable t) {
+            throw new AssertionError(t);
+        }
+    }
+
+    public String getTitle() {
+        return invoke(GET_TITLE);
+    }
+
+    @SuppressWarnings("unchecked")
+    private <T> T invoke(MethodHandle method) {
+        try {
+            return (T) method.invoke(view);
+        } catch (RuntimeException e) {
+            throw e;
+        } catch (Throwable t) {
+            throw new AssertionError(t);
+        }
+    }
+
+    @SuppressWarnings("unchecked")
+    private <T> T invoke(MethodHandle method, int slot) {
+        try {
+            return (T) method.invoke(view, slot);
         } catch (RuntimeException e) {
             throw e;
         } catch (Throwable t) {

